doc/releases/changelog-dev.md

```diff
@@ -73,6 +89,9 @@
 * Python 3.7 support is no longer maintained.
   [(#3276)](https://github.com/PennyLaneAI/pennylane/pull/3276)
 
+* Removed `qml.utils.decompose_hamiltonian()`, please use `qml.pauli_decompose()` instead.
+  [(#3384)](https://github.com/PennyLaneAI/pennylane/pull/3384)
```

Comment:
In the deprecations page you mention that this method is deprecated, but here you say it is removed. Which one is it?

Reply:
The method is removed, but shouldn't we deprecate it instead?

Reply:
I think we decided not to deprecate it, but directly move and update it. Its a smaller method which as Josh  mentioned is in demand, but users have a hard time finding it. So we don't expect removing it out right to cause too many issues.

Reply:
Updated deprecations page to say removed instead of deprecated 👍🏼